docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -4866,30 +4866,134 @@ Result:
 └───────┴───────────────┴──────┴──────────────┴──────────────┴──────────────────────┘
 ```
 
-## toInterval(Year\|Quarter\|Month\|Week\|Day\|Hour\|Minute\|Second)
+## toIntervalYear
 
-Converts a Number type argument to an [Interval](../data-types/special-data-types/interval.md) data type.
+Returns an interval of `n` years of data type [IntervalYear](../data-types/special-data-types/interval.md).
 
 **Syntax**
 
 ``` sql
-toIntervalSecond(number)
-toIntervalMinute(number)
-toIntervalHour(number)
-toIntervalDay(number)
-toIntervalWeek(number)
-toIntervalMonth(number)
-toIntervalQuarter(number)
-toIntervalYear(number)
+toIntervalYear(n)
 ```
 
 **Arguments**
 
-- `number` — Duration of interval. Positive integer number.
+- `n` — Number of years. Positive integer number. [Int*](../data-types/int-uint.md).
 
 **Returned values**
 
-- The value in `Interval` data type.
+- Interval of `n` years. [IntervalYear](../data-types/special-data-types/interval.md).
+
+**Example**
+
+Query:
+
+``` sql
+WITH
+    toDate('2019-01-01') AS date,
+    toIntervalYear(1) AS interval_to_year
+SELECT date + interval_to_year
+```
+
+Result:
+
+```response
+┌─plus(date, interval_to_year)─┐
```

Comment:
Let's add `AS result` in l. 4895 to get rid of the unaesthetic implicit type name.